core/themes/basis/template.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Implements hook_preprocess_page()'
+ * To add a class 'page-node-N' to each page.
+ */
+function basis_preprocess_page(&$variables) {
+  $node = menu_get_object();
+
+  // Add normalize.css from core as high up as possible in cascade
+  backdrop_add_css('core/misc/normalize.css', array(
+    'every_page' => true,
+    'group' => CSS_SYSTEM,
+    'weight' => -1000,
+  ));
```

Comment:
This should already be the case, perhaps we could separate this out into a core patch rather than handling it in Basis.